Regras.cs

```diff
@@ -0,0 +1,160 @@
+using System.Collections.Generic;
+using System.Linq;
+
+public class Regras
+{
+    #region Regras
+    //Carta Alta: A carta de maior valor. #10
+    public Carta CartaAlta(List<Carta> cartasMao)
+    {
+        return cartasMao.FirstOrDefault(c => c.Valor == cartasMao.Max(v => v.Valor));
+    }
+
+    //Um Par: Duas cartas do mesmo valor. #20
+    public int UmPar(List<Carta> cartasMao)
+    {
+        return 0;
+    }
+
+    //Dois Pares: Dois pares diferentes. #30
+    public int DoisPares(List<Carta> cartasMao)
+    {
+        return 0;
+    }
+
+    //Trinca: Três cartas do mesmo valor e das de valores diferentes. #40
+    public int Trinca(List<Carta> cartasMao)
+    {
+        foreach(var carta in cartasMao)
+        {
+            var qtndMesmoValor = cartasMao.Select(c => c.Valor = carta.Valor).ToList().Count();
+            if(qtndMesmoValor > 3)
+            {
+                return 40;   
+            }
+        }
+        
+        return 0;
+    }
+
+    //Straight (sequência): Todas as cartas com valores consecutivos. #50
+    public int Straight(List<Carta> cartasMao)
+    {
+        return 0;
+    }
+
+    //Flush: Todas as cartas do mesmo naipe. #60
+    public int Flush()
+    {
+        return 60;
+    }
+
+    //Full House: Uma trinca e um par. #70
+    public int FullHouse(List<Carta> cartasMao)
+    {
+        return 0;
+    }
+
+    //Quadra: Quatro cartas do mesmo valor. #80
+    public int Quadra(List<Carta> cartasMao)
+    {
+        return 0;s
```

Comment:
Essa linha aqui que está com erro. Por isso deu erro no azure.
```suggestion
        return 0;
```